### ISSUE TITLE
``prop.createSent`` creates broken E2s in certain cases

### ISSUE THREAD

**Issue by Vurv78** [thegrb93/StarfallEx#1215]
Was investigating a bug @Cheatoid encountered and found that since SF overrides the string library at runtime, E2's tokenizer is broken in https://github.com/wiremod/wire/blob/384809700d1cd250cf0f3f5b98d1e642e5131cd3/lua/entities/gmod_wire_expression2/base/tokenizer.lua#L91 (called by https://github.com/wiremod/wire/blob/384809700d1cd250cf0f3f5b98d1e642e5131cd3/lua/entities/gmod_wire_expression2/base/tokenizer.lua#L128)

This errors with "error in error handling" (thanks lua, very helpful), but really is just ``SF.Throw`` with "2 ext search length too long (500 max)" if you have a comma in the code it seems.

Could just push a change in wiremod to use string.find instead of ``(""):find()`` but this really pertains Starfall so I wanna know what you think @thegrb93

Another thing that is kinda gross is that the E2 acts as if it's been duped() and doesn't run with the first var set so first() never triggers which could be misleading

**Comment by thegrb93** [comment]
I think changing it to string.find is the only way to fix it.

As for first(), I thought duped E2 in general don't run first(), idk if that was ever changed or not.

**Comment by Cheatoid** [comment]
After further investigation into this, I think I came up with an actual fix for a series of errors encountered on this experiment.
It is Starfall at fault here, not Wiremod; Changing the above mentioned code in Wiremod codebase does not fully fix the issue, only a small subset of it. In other words I am still able to run into other errors, including the wild `error in error handling` still, even after applying the above patch into Wiremod.
Therefore, E2 spawning should be considered a **special case**. I will PR a very simple fix I came up with, it also makes the spawned E2 have `first()` evaluated as true (which I personally think is a good thing and correct behavior after all).
